ui/app/pages/swaps/actionable-message/actionable-message.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { renderWithProvider } from '../../../../../test/jest';
+import ActionableMessage from '.';
+
+describe('ActionableMessage', () => {
+  const createProps = (customProps = {}) => {
```

Comment:
Is there any reason that this method is created inside the describe block?

Reply:
No reason, I will move it outside.

Reply:
Done in all tests.